redisinsight/ui/src/pages/workbench/components/wb-view/WBViewWrapper.tsx

```diff
@@ -134,10 +136,12 @@ const WBViewWrapper = () => {
     multiCommands: string[] = [],
   ) => {
     const { connectionType, host, port } = state.instance
+    const mode = localStorageService.get(BrowserStorageItem.workbenchMode)
```

Comment:
you have several places where you get this mode from localStorage. Please, move it to one place.